gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,6 @@
-#Wed Aug 17 09:08:57 EEST 2016
+#Tue Nov 29 20:25:18 EET 2016
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-2.14.1-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-3.2.1-bin.zip
```

Comment:
I see some warnings from Gradle on Travis build https://travis-ci.org/vilnius/tvarkau-vilniu/builds/179856849 but these due to deprecation so it should be ok